content/applications/inventory_and_mrp/inventory/management/products/reordering_rules.rst

```diff
@@ -0,0 +1,126 @@
+================
+Reordering rules
+================
+
+Reordering rules are used to keep forecasted stock levels above a certain threshold without
+exceeding an upper limit. This is accomplished by specifying a minimum quantity that stock should
+not fall below and a maximum quantity that stock will be replenished up to.
+
+Reordering rules can be configured for each product based on the route used to replenish it. If a
+product uses the *Buy* route, then a Request for Quotation (RFQ) will be created when the reordering
+rule is triggered. If a product uses the *Manufacture* route, then a Manufacturing Order (MO) will
+be created instead. This will be the case for whichever replenishment route is selected.
+
+Configure products for reordering rules
+=======================================
+
+In order to use reordering rules for a product, it must first be correctly configured. Begin by
+navigating to :guilabel:`Inventory --> Products --> Products`, then select an existing product or
+create a new one by clicking :guilabel:`Create`.
+
+First, on the :guilabel:`General Information` tab, make sure that the :guilabel:`Product Type` is
+set to :guilabel:`Storable Product`. This is necessary because Odoo only tracks stock quantities for
+storable products, and this number is used to trigger reordering rules.
```

Comment:
```suggestion
First, on the :guilabel:`General Information` tab, make sure the :guilabel:`Product Type` is set to
:guilabel:`Storable Product`. This is necessary because Odoo only tracks stock quantities for
storable products, and this number is used to trigger reordering rules.
```